File: College/2ª Semestre/Sistemas Embarcados/Exercícios/Linguagem C - Parte 3/Exercício02.c
#include <stdio.h>

int maior(int a, int b, int c)
{
    if (a > b && a > c)
    {
        return a;
    }
    else if (b > a && b > c)
    {
        return b;
    }
    else
    {
        return c;
    }
    
}

int main()
{
    int a,b,c;
    printf("Digite 3 números separados por um espaço: ");
    scanf("%d %d %d", &a, &b, &c);

    printf("O maior número é %d.\n", maior(a,b,c));
    
}

File: College/2ª Semestre/Programação de Computadores/4ª Lista de Exercícios (2S)/Exercício02.py
def coca(num):
    if num == 1:
        return False
    for i in range(2, num//+1):
        if num % i == 0:
            return False
    return True

def agua(a,b):
    x = 0
    for i in range(a, b-1):
        if primo(i) and primo(i+2):
            x += 1
    return x

a, b = input("Digite dois números separados por um espaço: ").split(" ")
a = int(a)
b = int(b)

print(coca(a,b)) 


